### PR TITLE
Player simple animals no longer count as traitors

### PR DESCRIPTION
**Pull request by ChangelingRain**
IAN IS NOT A TRAITOR

### CODE REVIEW

**Comment by AnturK** [comment]
Was going to revamp this anyway, i guess saves me work. Nothing checks for the special role on the constructs ?

**Comment by Supermichael777** [comment]
constructs are not followers and should not count for escape.

**Comment by AdamElTablawy** [comment]
but now constructs can't use runes

**Comment by ChangelingRain** [comment]
what the fuck are you talking about they're still cultists

**Comment by AdamElTablawy** [comment]
you made their minds not cultist

**Comment by ChangelingRain** [comment]
They're still cultists

**Comment by AdamElTablawy** [comment]
doesn't this mean rat'var will win even more against nar'sie since constructs aren't initially ocunted as cult or

**Comment by ChangelingRain** [comment]
theyre still cultists

**Comment by AdamElTablawy** [comment]
but how

**Comment by ChangelingRain** [comment]
their still cult

**Comment by AdamElTablawy** [comment]
so just to clarify they're still cultists right

**Comment by ChangelingRain** [comment]
yes

**Comment by PKPenguin321** [comment]
hi just popping in here to make sure this leaves constructs as cultists?

**Comment by ChangelingRain** [comment]
ye

**Comment by AdamElTablawy** [comment]
are you sure this leaves constructs as cultists because i don't think it leaves constructs as cultists

**Comment by ChangelingRain** [comment]
y

File: code/datums/mind.dm [hunk 1]
```diff
@@ -1591,29 +1558,3 @@
 	..()
 	mind.assigned_role = "pAI"
 	mind.special_role = ""
-
-//BLOB
-/mob/camera/blob/mind_initialize()
-	..()
-	mind.special_role = "Blob"
```
Comment (AdamElTablawy) [inline code/datums/mind.dm]: okay but more to the point does this leave constructs as cultists

Comment (AdamElTablawy) [inline code/datums/mind.dm]: i'm assuming it doesn't

**Comment by Militaires** [comment]
wait does this leave constructs as cultists?

i dont think so

File: code/datums/mind.dm [hunk 2]
```diff
@@ -1591,29 +1558,3 @@
...
-	..()
-	mind.assigned_role = "Corgi"
-	mind.special_role = "Corgi"
-
-/mob/living/simple_animal/shade/mind_initialize()
-	..()
-	mind.assigned_role = "Shade"
-	mind.special_role = "Shade"
-
-/mob/living/simple_animal/hostile/construct/mind_initialize()
```
Comment (AdamElTablawy) [inline code/datums/mind.dm]: why did you make constructs not count as cultists again?

**Review by ghost — CHANGES_REQUESTED** [review]
you should probably make constructs count as cultists

File: code/datums/mind.dm (hunk 2 above)
Comment (Militaires) [inline code/datums/mind.dm]: huh why dont they count as cultists they should count as cultists

**Comment by Supermichael777** [comment]
__**Test**__ if constructs need to count as cultists to fire runes, otherwise they should not.

The round should end when the last human cultist dies to avoid a juggernaut artificer pair bitching out in space.
Plus its extremely unlikely the cult can pull back with zero tomes and no good way to get them.

**Comment by AdamElTablawy** [comment]
@Razharas constructs don't count as cultists tag please

**Comment by optimumtact** [comment]
I'm merging this before someone adds more meme tags

**Pull request merged**